pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -258,17 +292,25 @@ func (l *listUsersQuery) expandRewrite(
 			})
 		}
 
-		return pool.Wait()
+		err = pool.Wait()
 	default:
 		panic("unexpected userset rewrite encountered")
 	}
+
+	if err != nil {
+		telemetry.TraceError(span, err)
+		return err
+	}
+	return nil
 }
 
 func (l *listUsersQuery) expandDirect(
 	ctx context.Context,
 	req *internalListUsersRequest,
 	foundUsersChan chan<- *openfgav1.User,
 ) error {
+	ctx, span := tracer.Start(ctx, "expandDirect")
+	defer span.End()
 	typesys, err := l.typesystemResolver(ctx, req.GetStoreId(), req.GetAuthorizationModelId())
 	if err != nil {
```

Comment:
`telemetry.TraceError(span, err)`?

Reply:
It's just not ever clear where or if the error is being traced. Nothing actionable though.

Reply:
The error will be reported here:

```go
listUsersQuery := listusers.NewListUsersQuery(s.datastore,
		listusers.WithListUsersQueryLogger(s.logger))
	resp, err := listUsersQuery.ListUsers(ctx, req)
	if err != nil {
		telemetry.TraceError(span, err)
		return nil, err
	}
```

Reply:
My point is, its seemingly arbitrary wether it goes inside the function (which `typesystemResolver ` does) or reads the output (above example). But its really not a big deal.